index.js

```diff
@@ -49,67 +50,68 @@ client.on("message", function (message) {
     inventoryCommand(message);
   } else if (message.content.startsWith("!lookup")) {
     lookUpCommand(message);
+  } else if (message.content.startsWith("!remove")) {
+    removeCardCommand(message);
   }
 });
 
 const drawCommand = (message) => {
-  var ref = db.ref("users");
+  const ref = db.ref("users");
   ref.once("value", function (snapshot) {
     const userData = snapshot.val();
+    let userID;
+    let userInventory;
     let userDraw = message.author.id;
     let user = userData[userDraw];
```

Comment:
by the way, you already have variables up here that contain the userId and the user object, they should be `const`s and you can re-use them